bootstrap/config/config.go

```diff
@@ -19,13 +19,15 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"github.com/edgexfoundry/go-mod-bootstrap/v2/config"
 	"io/ioutil"
 	"math"
 	"reflect"
 	"sync"
 	"time"
 
+	"github.com/BurntSushi/toml"
```

Comment:
Not clear to me how replacement of pelletier/go-toml with BurntSushi/toml had not collateral changes.

Reply:
Not sure where that came from

Reply:
Fixed